src/marqo/s2_inference/clip_utils.py

```diff
@@ -341,10 +340,48 @@ def encode_image(self, images: Union[str, ImageType, List[Union[str, ImageType]]
         return self._convert_output(outputs)
 
 
-
-
-
-
+def get_multilingual_clip_properties() -> Dict:
```

Comment:
is it good practice to have `get_multilingual_clip_properties` in `clip_utils` instead of `model_registry` like the others? would we eventually want to move, let's say `_get_onnx_clip_properties` to `onnx_clip_utils` as well?

Reply:
It is nice that they were centralised in the model_registry. This move was necessary for Marqo to run properly to avoid a circular import error. 

In the future, there are a few ways to split it out. 
Option 1. We can put the model load mappings into a separate file: https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/s2_inference/model_registry.py#L1565
Option 2. We can make the properties associated with model class